### PR TITLE
status/2024q1/clusteradm: Add report

### PR DESCRIPTION
**Pull request by dbaio**
@ppaeps @lwhsu

### CODE REVIEW

File: website/content/en/status/report-2024-01-2024-03/clusteradm.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Set up a new mirror in Chicago.
+More details are provided https://freebsdfoundation.org/blog/powering-up-the-future-the-new-freebsd-cluster-in-chicago/[here].
```
Comment (pauamma) [inline website/content/en/status/report-2024-01-2024-03/clusteradm.adoc:15]:
```suggestion
* https://freebsdfoundation.org/blog/powering-up-the-future-the-new-freebsd-cluster-in-chicago/[Set up a new mirror in Chicago].
```

Better out-of-main-text link text accessibility (some screenreader users do that).

File: website/content/en/status/report-2024-01-2024-03/clusteradm.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* https://www.isc.org/[Internet Systems Consortium]
+* https://www.inx.net.za/[INX-ZA]
+* https://www.kddi-webcommunications.co.jp/english/[KDDI Web Communications Inc]
+* https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
+* https://www.metapeer.com/[MetaPeer]
+* https://www.nyi.net/[New York Internet]
+* https://nic.br/[NIC.br]
+* https://www.teleservice.net/[Teleservice Skåne AB] (new since 2023Q4)
+* https://your.org/[Your.Org]
+
```
Comment (lwhsu) [inline website/content/en/status/report-2024-01-2024-03/clusteradm.adoc:37]: How about adding something like "Please contact us if you want to provide an official mirror."

**Review by lwhsu — APPROVED** [review]
Thanks very much!

File: website/content/en/status/report-2024-01-2024-03/clusteradm.adoc (hunk 2 above)
Comment (dbaio) [reply website/content/en/status/report-2024-01-2024-03/clusteradm.adoc:36]: done

**Review by pauamma — APPROVED** [review]

**Comment by lsalvadore** [comment]
Merged, thanks. I have only added a few `link:` macro tags. Although most of the times it is unnecessary to put them explicitly, we want them in the FreeBSD doc repository (they are needed for some particular locales I think).

**Pull request closed without merging**